src/MainOpt.h

```diff
@@ -48,6 +48,9 @@ struct MainOpt {
   /// Can optionally use the `graylog_logger` library to log to this address.
   uri::URI GraylogLoggerAddress;
 
+  uri::URI GrafanCarbonAddress;
```

Comment:
Typo, "GrafanCarbonAddress" -> "GrafanaCarbonAddress". But maybe you meant "GraphiteCarbonAddress" anyway? I would suggest just "CarbonAddress".

Reply:
The intent was `GrafanaCarbonAddress`. I suspect that most people using Grafana does not know what Graphite is and that even fewer knows what Carbon is. Hence picking that name instead of "CarbonAddress".

Reply:
If they know about Grafana and not Carbon won't this just lead to them putting the wrong address in the field? I'm a little uneasy about linking it to Grafana when one or more facilities may end up using Grafana with a different database, or Carbon with a different dash.
PSI were using InfluxDB, I don't know if that is still the case.

Reply:
Based on a comment from Mark last week, I believe that they do not use Grafana at all. Now, your comment is fair but I do not agree with it. So: @mattclarke, can we get a tiebreaker here?